### PR TITLE
Branding

### PR DESCRIPTION
**Pull request by StevenLocke**
@Calligre/frontend 

Merging the branding page changes into the organizer branch.  There are some weird commits here, I think I branched off of something I shouldn't have.

Some of these changes are already on the organizer branch, like Nav.jsx, and others are just linting.

The important changes are Branding.jsx, BrandStore.jsx, and Info.jsx which changed to use the proper AuthService.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,16 +6,18 @@
   "dependencies": {
     "auth0-lock": "10.11.0",
     "babel-core": "6.22.1",
-    "babel-loader": "6.2.10",
+    "babel-eslint": "7.1.0",
```
Comment (ericljpemberton) [inline package.json]: move to devDependencies

File: package.json [hunk 2]
```diff
@@ -6,16 +6,18 @@
...
+    "babel-loader": "6.2.0",
```
Comment (ericljpemberton) [inline package.json]: Change back to 6.2.10?

File: package.json [hunk 3]
```diff
@@ -6,16 +6,18 @@
...
     "babel-plugin-react-html-attrs": "2.0.0",
     "babel-plugin-transform-class-properties": "6.22.0",
     "babel-plugin-transform-decorators-legacy": "1.3.4",
     "babel-preset-es2015": "6.22.0",
     "babel-preset-react": "6.22.0",
     "babel-preset-stage-0": "6.22.0",
-    "components": "^0.1.0",
+    "components": "0.1.0",
     "css-loader": "0.23.1",
+    "eslint-import-resolver-webpack": "0.7.0",
```
Comment (ericljpemberton) [inline package.json]: devDependency

File: src/js/pages/attendee/Info.jsx [hunk 4]
```diff
@@ -1,57 +1,64 @@
...
   render() {
-    const {logo, confName, location, starttime, endtime, other, twitter, facebook} = this.state;
+    const { logo, conf_name, location, starttime, endtime, other, twitter, facebook } = this.state;
     return (
       <div id="conference-info">
-        <img class="logo" src={logo} />
-        <h3 class="other">{other}</h3>
-        <div class="info-container">
-          <h1>You're at: {confName}</h1>
+        <img className="logo" alt="Logo" src={logo} />
```
Comment (larajanecka) [inline src/js/pages/attendee/Info.jsx:53]: psht supporting screen readers, who does that?

File: src/js/pages/attendee/Layout.jsx [hunk 5]
```diff
@@ -10,7 +10,7 @@ export default class Layout extends React.Component {
 
     return (
       <div>
-        <Nav location={location}/>
+        <Nav location={location} auth={this.props.route.auth}/>
```
Comment (larajanecka) [inline src/js/pages/attendee/Layout.jsx]: I don't think this is necessary?

File: src/js/pages/organizer/Branding.jsx [hunk 6]
```diff
@@ -1,9 +1,122 @@
...
+
+    BrandStore.on("receivedBranding", this.setBranding);
+    BrandStore.on("savedBranding", this.brandingSaved)
+    BrandStore.getBranding();
+  }
+
+  setBranding = () => {
+    let branding = BrandStore.branding;
+    
+    Object.keys(branding).forEach(function(key) {
```
Comment (larajanecka) [inline src/js/pages/organizer/Branding.jsx]: Why the loop? Wouldn't it be easier to just do it explictily?

File: src/js/pages/organizer/Branding.jsx [hunk 7]
```diff
@@ -1,9 +1,122 @@
...
+    console.log("Branding saved successfully.");
+  }
+
+  render() {
+    return (
+      <div>
+        <table width="100%" height="100%">
+          <tbody>
+            <tr>
+              <td width="50%">
```
Comment (larajanecka) [inline src/js/pages/organizer/Branding.jsx]: no inline style, clarisse will murder you

**Review by larajanecka — APPROVED** [review]
Technically good to merge, but if you do I would fear for your life.

File: src/js/pages/attendee/Info.jsx (hunk 4 above)
Comment (StevenLocke) [reply src/js/pages/attendee/Info.jsx:53]: Right, who needs the visually impaired?

File: src/js/pages/organizer/Branding.jsx (hunk 7 above)
Comment (StevenLocke) [reply src/js/pages/organizer/Branding.jsx]: This is, I think, necessary. It's a table with two elements, each having half the full width.

I could remove it, but I think it'll have an undefined, maybe half, maybe not, look to it.

**Comment by StevenLocke** [comment]
@larajanecka Addressed the things, take a quick one-over again please?

File: src/js/pages/attendee/Info.jsx (hunk 4 above)
Comment (larajanecka) [reply src/js/pages/attendee/Info.jsx:53]: yeah, lets just kill 'em

**Pull request merged**